capi/VideoDecoderCapi.h

```diff
@@ -19,16 +19,10 @@
  *  Boston, MA 02110-1301 USA
  */
 
-#ifndef __VIDEO_DECODER_CAPI_H__
-#define __VIDEO_DECODER_CAPI_H__
+#ifndef videodecodercapi_h
+#define videodecodercapi_h
```

Comment:
I prefer the previous header guard naming syntax... but it appears both styles are used throughout this project.  Question is, which style is preferred in yami?  It would be better to enforce only one policy.